scripts/download_elevation_grid.py

```
@@ -0,0 +1,484 @@
+    def _estimate_generic(self, lat: float, lon: float) -> float:
+        """Generic elevation estimation."""
+        base = 100
+        variation = np.sin(lat * 10) * 50 + np.cos(lon * 10) * 30
+        return max(0, base + variation)
+    
+    def _is_coastal(self, lat: float, lon: float, country: str) -> bool:
+        """Simple coastal detection."""
+        # This is simplified - in reality would use shapefile
+        if country == 'JP':
+            return lon < 130 or lon > 140 or lat < 30
+        elif country == 'IN':
```

Ruff (0.12.2)

Unnecessary `elif` after `return` statement

Remove unnecessary `elif`

(RET505)